nncf/common/graph/transformations/commands.py

```diff
@@ -31,14 +31,19 @@ class TransformationType(OrderedEnum):
 
 
 class TargetType(OrderedEnum):
-    LAYER = 0
-    BEFORE_LAYER = 1
-    AFTER_LAYER = 2
-    PRE_LAYER_OPERATION = 3
-    POST_LAYER_OPERATION = 4
-    OPERATION_WITH_WEIGHTS = 5
-    OPERATOR_PRE_HOOK = 6
-    OPERATOR_POST_HOOK = 7
+    # Definitions: TF-nodes - visible in TF-graph, TF-operations - invisible in TF-graph
+    # Both TF-nodes and TF-operations may be used for hooking in TF.
+    LAYER = 0  # add or remove a TF-node in a TF-graph
+    BEFORE_LAYER = 1  # additional TF-node in TF-graph before TF-layer op
+    AFTER_LAYER = 2  # additional TF-node in TF-graph after TF-layer op
+
+    # These three add a TF-operation in TF, not a TF-node. In PT, these map to module pre- and post-ops.
+    PRE_LAYER_OPERATION = 3  # A pre-operation that can access attrs of the associated PT-module or TF-layer
+    POST_LAYER_OPERATION = 4  # A post-operation that can access attrs of the associated PT-module or TF-layer
+    OPERATION_WITH_WEIGHTS = 5  # Same as PRE_LAYER_OPERATION, but targets weights of the layer/module specifically
+
+    OPERATOR_PRE_HOOK = 6  # A pre-operation in PT that does not in general access module attrs - N/A in TF
+    OPERATOR_POST_HOOK = 7  # A post-operation in PT that does not in general access module attrs - N/A in TF
```

Comment:
It looks like these comments are not a docstring and could not be added in the documentation automatically way if we will decide to do it. I mean `TargetType.__doc__` does not have any description. I can suggest adding a description as the docstring  at the beginning of the class:
```
class TargetType(OrderedEnum):
    """
    Enumeration of the target point types.

    Target point types define the meaning and operations that can be performed
    on the target point in the model graph.

    `LAYER` - add, remove or update a TF Keras Layer.  Supports only in the TF-backend.
    ....
    """
```
Of course, you can add a docstring for each type, but, as I know, It requires additional implementation. 

Could you add a description for other enumerations in the commands.py.

Reply:
Done